pkg/reconciler/taskrun/taskrun.go

```diff
@@ -727,8 +662,7 @@ func updateStoppedSidecarStatus(pod *corev1.Pod, tr *v1beta1.TaskRun, c *Reconci
 			})
 		}
 	}
-	_, err := c.updateStatus(tr)
```

Comment:
Just making a note here in case this comes into question, but checked this out and seems to be fine without the call to `updateStatus`. Thanks for noticing and correcting.